### PR TITLE
Add __get__ to the column and relationship to improve usage

### LINKED ISSUE #170: Supporting __get__ and __set__ on Column object
The old sqlalchemy-stubs from dropbox included an approximation for the behavior of mapped classes using declarative setup, that is

```
class Foo(Model):
  a = Column(DateTime)

Foo.a
# type is Column[DateTime]
Foo().a
# type is datetime
```
They did so by defining the __get__ method on Column here: https://github.com/dropbox/sqlalchemy-stubs/blob/master/sqlalchemy-stubs/sql/schema.pyi#L131

Maybe the same can be done for __set__, so that

```
Foo().a = 3
# wrong type, expected datetime object
```

Of course the whole thing is supported already when using the mypy plugin, but for example when using pylance (vscode type engine) which does not support plugins, it gets confused when using and assigning object values as mapped columns and complains about mismatching types.

I think the same is done on relationship() property

I'm unsure of the deeper implications of adding such type hints, though

### PR DESCRIPTION
**Pull request by CaselIT**
when the plugin is not supported, for example when using other type checkers.

### Description
This was taken from the dropbox stubs.

Fixes #170

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [x] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: test/files/orm_columns.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from typing import List
```
Comment (zzzeek) [inline test/files/orm_columns.py:1]: this gets tested with the plugin installed, so isnt really tested, right?

Comment (CaselIT) [reply test/files/orm_columns.py:1]: yes, but the objective it to ensure that with the plugin nothing unsurprising happens.

I've looked at it in vscode and the types are inferred correctly

**Comment by CaselIT** [comment]
There is also  column_property(), deferred(), query_property(), etc

**Comment by CaselIT** [comment]
closing since for v2 we went on another direction

**Pull request closed without merging**